assets/js/translator.js

```diff
@@ -447,12 +447,33 @@ function getPairs() /*: JQueryPromise<any> */ {
             }
         });
 
-        for(var srcLang in pairs) {
-            // Default for new users is first available pair
-            curSrcLang = srcLang;
-            curDstLang = pairs[srcLang][0];
-            break;
+        // Default for new users is first available Browser Preference Language pair
+        var brwsLangs = navigator.languages; // Chrome, Mozilla and Safari
+        var prefLang;
+        $.each(brwsLangs, function(index) {
```

Comment:
`$.each` also gives you the actual element, using that should work fine here.

Reply:
already $.each is used. I dont understand?

Reply:
The callback receives each value, there's no need to re-index the values from the array. Please look at the documentation.

Reply:
Aah okay okay. :)

Reply:
Gotcha! :)

Reply:
Done. :)